tests/bluetooth/tester/src/bttester.h

```diff
@@ -238,6 +250,35 @@ struct gap_conn_param_update_cmd {
 	u16_t timeout;
 } __packed;
 
+#define GAP_PAIRING_CONSENT		0x17
+struct gap_pairing_consent_cmd {
+	u8_t address_type;
+	u8_t address[6];
+	u8_t consent;
+} __packed;
+
+#define GAP_OOB_LEGACY_SET_DATA		0x18
+struct gap_oob_legacy_set_data_cmd {
+	u8_t oob_data[16];
+} __packed;
+
+#define GAP_OOB_SC_GET_LOCAL_DATA	0x19
+struct gap_oob_sc_get_local_data_rp {
```

Comment:
I know this is correct according to the BTP spec, but don't we need to have the RPA address as well?
If you see `struct bt_le_oob` we have address, random and confirm.

Or do the PTS test connect using a different method?

Reply:
That should be changed in the BTP spec first, so please raise the issue there.

Reply:
Will do.